engine/src/main/java/org/hibernate/validator/resourceloading/PlatformResourceBundleLocator.java

```diff
@@ -149,7 +149,12 @@ public PlatformResourceBundleLocator(String bundleName,
 		if ( !localesToInitialize.isEmpty() ) {
 			Map<Locale, ResourceBundle> tmpPreloadedResourceBundles = CollectionHelper.newHashMap( localesToInitialize.size() );
 			for ( Locale localeToPreload : localesToInitialize ) {
-				tmpPreloadedResourceBundles.put( localeToPreload, doGetResourceBundle( localeToPreload ) );
+				ResourceBundle bundle = doGetResourceBundle( localeToPreload );
+				if ( bundle == null ) {
+					LOG.resourceBundleNotPreLoaded( localeToPreload );
+					continue;
+				}
+				tmpPreloadedResourceBundles.put( localeToPreload, bundle );
```

Comment:
had to do this, as with the change to `Map.copyOf` `null`s are not allowed anymore and the startup would fail ... (noticed it when running the benchmarks)